pkg/importer/dataStream.go

```diff
@@ -183,6 +212,38 @@ func (d dataStream) http() (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
+func (d dataStream) glance() (io.ReadCloser, error) {
+    opts := gophercloud.AuthOptions{
+        IdentityEndpoint: d.provider.IdentityEndpoint,
+        Username: d.accessKeyId,
+        Password: d.secretKey,
+        DomainName: d.provider.Domain,
+      }
```

Comment:
indentation here and below. Try running `gofmt -w -s`

Reply:
Done